spec/features/views_games_list_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+
+
+feature 'user views restaurants' do
```

Comment:
Delete the blank lines and change the name from games to restaurants. Be careful when copying/pasting code.